samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs

```diff
@@ -55,16 +55,17 @@ protected override Task<QnAMakerOptions> GetQnAMakerOptionsAsync(DialogContext d
 
         protected async override Task<QnADialogResponseOptions> GetQnAResponseOptionsAsync(DialogContext dc)
         {
-            var noAnswer = (Activity)Activity.CreateMessageActivity();
-            noAnswer.Text = this._configuration["DefaultAnswer"] ?? DefaultNoAnswer;
+            var defaultAnswerActivity = (Activity)Activity.CreateMessageActivity();
+            var defaultAnswerFromConfig = this._configuration["DefaultAnswer"];
```

Comment:
If this is from default, shouldn’t DefaultAnswer be a public readonly that is set with the configuration value if it exists? Is the const worth having the ternary check?

Reply:
Made changes as suggested.  Keeping the check for whitespace as Default answer cannot be whitespace or empty.